UnitTests/Tests/FISAppDelegateSpec.m

```diff
@@ -27,43 +27,41 @@
         __block NSMutableArray *originalArray;
 
         beforeEach(^{
-            originalArray = [@[ @1, @2, @3 ] mutableCopy];
+            originalArray = [@[ @"Hello", @"this", @"is" ] mutableCopy];
         });
 
         it(@"must not modify the original array", ^{
-            [delegate arrayByAddingString:@"hello" toArray:originalArray];
-            expect(originalArray).to.equal(@[ @1, @2, @3 ]);
+            [delegate arrayByAddingString:@"Patrick" toArray:originalArray];
+            expect(originalArray).to.equal(@[ @"Hello", @"this", @"is" ]);
         });
 
         it(@"should actually return a mutable array", ^{
-            NSMutableArray *newArray = [delegate arrayByAddingString:@"hello" toArray:originalArray];
+            NSMutableArray *newArray = [delegate arrayByAddingString:@"Patrick" toArray:originalArray];
             expect(newArray).to.beKindOf([NSMutableArray class]);
         });
 
-        it(@"should append the given string to the array", ^{
-            NSMutableArray *newArray = [delegate arrayByAddingString:@"hello" toArray:originalArray];
-            expect(newArray).to.equal(@[ @1, @2, @3, @"hello" ]);
+        it(@"should add the given string to the array", ^{
+            NSMutableArray *newArray = [delegate arrayByAddingString:@"Patrick" toArray:originalArray];
+            expect(newArray).to.equal(@[ @"Hello", @"this", @"is", @"Patrick" ]);
         });
     });
 
-    describe(@"all-caps counting method", ^{
-        it(@"should count all-caps strings", ^{
-            NSUInteger count = [delegate countOfStringsInAllCapsInArray:@[ @"OH", @"YES", @"WOOO" ]];
+    describe(@"uppercase string counting method", ^{
+        it(@"should count uppercase strings", ^{
+            NSUInteger count = [delegate countOfUppercaseStringsInArray:@[ @"OH", @"YES", @"WOOO" ]];
             expect(count).to.equal(3);
         });
 
         it(@"should ignore whitespace, numbers, and punctuation", ^{
-            NSUInteger count = [delegate countOfStringsInAllCapsInArray:@[ @"HELLO THERE", @", Y'ALL", @"YAH!" ]];
+            NSUInteger count = [delegate countOfUppercaseStringsInArray:@[ @"HELLO THERE", @", Y'ALL", @"YAH!" ]];
             expect(count).to.equal(3);
         });
 
         it(@"should not count lower-case or mixed-case strings", ^{
-            NSUInteger count = [delegate countOfStringsInAllCapsInArray:@[ @"oH No", @"what NoW", @"AAAH"]];
+            NSUInteger count = [delegate countOfUppercaseStringsInArray:@[ @"oH No", @"what NoW", @"AAAH"]];
             expect(count).to.equal(1);
         });
 
-        // Could write a test to ignore strings that consist entirely of non-alphabetical characters,
-        // but that's not what this lab is really about.
```

Comment:
I would probably leave this in... helpful reminder to the students about what is actually important